partiql-planner/src/main/kotlin/org/partiql/planner/internal/transforms/RexConverter.kt

```diff
@@ -325,24 +326,24 @@ internal object RexConverter {
             // Wrap if a NOT, if necessary
             return when (symbol) {
                 "<>", "!=" -> {
-                    val op = negate(call("eq", *args.toTypedArray()))
+                    val op = negate(call(FunctionUtils.OP_EQ, *args.toTypedArray()))
                     rex(type, op)
                 }
                 else -> {
                     val name = when (symbol) {
                         // TODO eventually move hard-coded operator resolution into SPI
-                        "<" -> "lt"
-                        ">" -> "gt"
-                        "<=" -> "lte"
-                        ">=" -> "gte"
-                        "=" -> "eq"
-                        "||" -> "concat"
-                        "+" -> "plus"
-                        "-" -> "minus"
-                        "*" -> "times"
-                        "/" -> "divide"
-                        "%" -> "modulo"
-                        "&" -> "bitwise_and"
+                        "<" -> FunctionUtils.OP_LT
```

Comment:
How about a "System.get(..)" pattern here rather than having a big list of hardcoded strings? Maybe it's still hardcoded strings, but at least packaged within something that could later be expanded upon.